CONTRIBUTING.md

```diff
@@ -132,9 +132,9 @@ blog post.
 #### Nightly tests
 
 After changes are integrated, every evening at 10pm EST (3am UTC), Slicer build bots (aka factories)
-will build, test and package Slicer application and all its extensions on Linux, MacOSX
-and Windows. Results are published daily on [CDash](http://slicer.cdash.org/index.php?project=Slicer4)
-and developers introducing changes introducing build or test failures are notified by
+will build, test and package the Slicer application and all its extensions on Linux, MacOSX
```

Comment:
MacOSX -> macOS

See https://en.wikipedia.org/wiki/MacOS

Reply:
Ok, I think I had kept because Slicer 4.10 and 4.11 (as of now) still supports 10.11 as the minimum which is OS X 10.11 and not macOS 10.11. Though once 4.11 uses Qt 5.12.X the minimum will be 10.12 which was a macOS version.

I had also looked into other OS X to macOS but most were CMake related which we can’t really do anything about.

Reply:
Fair point. Since we still support 10.11, that makes sense.